modules/nf-core/bedtools/shift/meta.yml

```diff
@@ -0,0 +1,37 @@
+name: bedtools_shift
+description: Shifts each feature by specific number of bases
+keywords:
+  - bed
+  - shiftBed
+tools:
+  - bedtools:
+      description: |
+        A set of tools for genomic analysis tasks, specifically enabling genome arithmetic (merge, count, complement) on various file types.
+      documentation: https://bedtools.readthedocs.io/en/latest/content/tools/slop.html
+      licence: ["MIT"]
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - bed:
+      type: file
+      description: Input BED file
+      pattern: "*.{bed}"
```

Comment:
Can you add the meta2 and the genome file here too?

Reply:
I can't find the other converstaion, but why is there a need for "meta2" if it's only a file of chr sizes?

Reply:
Some pipelines use multiple kinds of genomes and need a way to parse this information. This can be done through a meta field, so in this case it makes sense to add it :)